TodoApplication/Models/TodoListItem.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TodoListItem.swift
+//  TodoApplication
+//
+//  Created by Abhishek Kumar Sinha on 26/11/24.
+//
+
+import Foundation
+
+// MARK: IT CONTAINS THE ITEMS OF TASK LIST
+
+struct TodoListItem : Codable, Identifiable {
+    
+    let id : String
+    let title : String
+    let DueDate : TimeInterval
+    let CreatedDate : TimeInterval // when the user is created the list
+    var isDone : Bool  // for check and uncheck
```

Comment:
variable name should start with lowercase

Reply:
I will make sure to change the variable name to start with a lowercase letter as per the convention